client/extensions/woocommerce/app/order/order-activity-log/index.js

```diff
@@ -39,21 +39,30 @@ function getSortedEvents( events ) {
 	return eventsByDay;
 }
 
-class OrderNotes extends Component {
+class OrderActivityLog extends Component {
 	static propTypes = {
 		orderId: PropTypes.number.isRequired,
 		siteId: PropTypes.number.isRequired,
 	};
 
 	constructor( props ) {
```

Comment:
You can remove the constructor since it's not doing anything but running `super` now.